src/LiveApi.js

```diff
@@ -13,6 +13,8 @@ let WebSocket = typeof window !== 'undefined' ? window.WebSocket : MockWebSocket
 const shouldIgnoreError = error =>
     error.message.includes('You are already subscribed to') ||
         error.message.includes('Input validation failed: forget');
+const shouldNotThrow = json => 
+    json.error.code === 'SelfExclusion' && json.msg_type === 'authorize';
 
```

Comment:
@nuruddeensalihu i thought all errors return to frontend by default? do you mean return as data instead of error?

If so, why? cant we directly catch from caller?
It's good to keep error handling consistent in library so we dont surprise any library user